eng/pipelines/templates/steps/analyze.yml

```diff
@@ -29,6 +29,11 @@ steps:
       pwsh: true
     displayName: Spell check public API
 
+  - task: UsePythonVersion@0
```

Comment:
We should consider putting this as part of the verify-readmes.yml template with an opt-out if places we don't want to override the python version in the pipeline.